web_widget_one2many_tags/static/src/js/web_widget_one2many_tags.js

```diff
@@ -23,7 +24,7 @@ openerp.web_widget_one2many_tags = function(instance)
                     arrow: {
                         onArrowClick: function(e)
                         {
-                            var list_view = new instance.web.form.One2ManyListView(
+                            var list_view = new widget_one2many_tag.web.form.One2ManyListView(
```

Comment:
no. all of them need to be required at the top of the file, then used in the code. Won't repeat this for every instance, but you basically need to change this in every changed line of code in this file